stable/falco/values.yaml

```diff
@@ -199,6 +199,21 @@ integrations:
     aws_secret_access_key: ""
     aws_default_region: ""
 
+
+  # If GCloud Pub/Sub integration is enabled, falco will be configured to use this
+  # integration as file_output and sets the following values:
+  # * json_output: true
+  # * json_include_output_property: true
+  # * file_output:
+  #     enabled: true
+  #     keep_alive: true
+  #     filename: /var/run/falco/nats
+  pubsubOutput:
+    enabled: false
+    credentials_data: ""
```

Comment:
Follow naming conventions: https://helm.sh/docs/chart_best_practices/#naming-conventions

The AWS ones should be also camelCase, that was a violation that will be fixed soon.